package edu.example.learner_kotlin.courseabout.coursereview.exception


class ReviewTaskException(message: String?, val statusCode: Int) : RuntimeException(message)
